### PR TITLE
[Docs] Add `FairScale` CN version blog post.

### PR DESCRIPTION
**Pull request by keli-wen**
## Description

Add `FairScale` CN version blog post.

- [x] Explains what `FairScale` is.
- [x] Explains the `FairScale` usage in `Llama`. 
- [x] Explains the basic distributed communication in model parallel. 

## Checklist
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [x] I've leverage the `lint.sh` to beautify the python and c++ code.
- [x] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [x] Code is well-documented.

### CODE REVIEW

**Review by NANWOOD — COMMENTED** [review]
整体的排版需要商榷

File: docs/CN/FairScale/RowParallelLinear.png
Comment (NANWOOD) [inline docs/CN/FairScale/RowParallelLinear.png:1]: 第一行和第二行的 weight 以及 input 表示尺寸的 in feature / out feature 以及 num input / in feature 的箭头有点迷惑性

Comment (NANWOOD) [inline docs/CN/FairScale/RowParallelLinear.png:1]: 第三行的结果实际上应该是维度拼接，用+是否合理

File: docs/CN/FairScale/README.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+> - `output` 是 $n \times m$ 的输出矩阵。
+> - `output_grad` 是输出的梯度，维度是 $n \times m$ 。
+>
+> 根据链式法则，权重的梯度 `weight_grad` 的计算过程是：
+> $$
+> \texttt{weight\_grad} = \texttt{input}^T \cdot \texttt{output\_grad}
+> $$
+>  输入的梯度 `input_grad` 的计算过程是：
+> $$
+> \texttt{input\_grad} = \texttt{output\_grad} \cdot \texttt{weight}^T
```
Comment (NANWOOD) [inline docs/CN/FairScale/README.md:153]: render error.

Comment (keli-wen) [reply docs/CN/FairScale/README.md]: fixed

File: docs/CN/FairScale/RowParallelLinear.png
Comment (NANWOOD) [reply docs/CN/FairScale/RowParallelLinear.png:1]: Replace arrows with straight lines.

**Review by NANWOOD — APPROVED** [review]
或许可以在文档中说明 ColumnParallelLinear 和 RowParallelLinear 的效率比较.

File: docs/CN/FairScale/RowParallelLinear.png
Comment (keli-wen) [reply docs/CN/FairScale/RowParallelLinear.png:1]: Get, updated. Please kindly check it.

**Review by NANWOOD — APPROVED** [review]
Temporarily feasible.

File: docs/CN/FairScale/README.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+...
+# Used in the Attention model.
+fs_init.get_model_parallel_world_size()
+```
+
+接下来我将一步步介绍上述代码的作用。显然，我们可以分成两个模块：`model_parallel.initialize` 和 `model_parallel.layers`。由于官方文档并没有 `model_parallel` 相关的更新， 以下内容主要基于 Fair Scale 的 source code 及对其 comments 的理解。
+
+### `fairscale.nn.model_parallel.initialize`
+
+*Function* `initialize_model_parallel`
```
Comment (NANWOOD) [inline docs/CN/FairScale/README.md:46]: Function 可以考虑加粗吗

Comment (keli-wen) [reply docs/CN/FairScale/README.md]: Done.

**Pull request merged**